api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupResumoMensalDAHReport.java

```diff
@@ -20,7 +20,6 @@
 import org.openmrs.module.reporting.report.ReportDesign;
 import org.openmrs.module.reporting.report.definition.ReportDefinition;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
```

Comment:
Was this dependency removed because it was not being used or the code that was using it got removed? @TinoMuianga

Reply:
This action was taken subsequent to executing "mvn clean install," and I presume that the dependency was removed due to non-utilization. @ItsMurumba